src/execution/sql/functions/string_functions.cpp

```diff
@@ -363,4 +363,28 @@ void StringFunctions::Right(UNUSED_ATTRIBUTE exec::ExecutionContext *ctx, String
   }
 }
 
+void StringFunctions::Position(exec::ExecutionContext *ctx, Integer *pos, const StringVal &search_str,
+                               const StringVal &search_sub_str) {
+  if (search_str.is_null_ || search_sub_str.is_null_) {
+    *pos = Integer::Null();
+    return;
+  }
+
+  auto search_str_view = std::string(search_str.StringView());
```

Comment:
Instead of copying the string every time, use a custom find function: https://stackoverflow.com/a/19839371

Reply:
Done.